README.md

```diff
@@ -29,57 +26,43 @@ We appreciate your feedback and energy helping us improve our services. [Please
 - Create an Azure account with an active subscription. For details, see [Create an account for free](https://azure.microsoft.com/free/?WT.mc_id=A261C142F).
 - [Node.js (12.18.4 and above)](https://nodejs.org/en/download/)
 - [Visual Studio (2017 and above)](https://visualstudio.microsoft.com/vs/)
-- [.NET Core 3.1](https://dotnet.microsoft.com/download/dotnet-core/3.1) (Make sure to install version that corresponds with your visual studio instance, 32 vs 64 bit)
 - Create an Azure Communication Services resource. For details, see [Create an Azure Communication Resource](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/create-communication-resource). You'll need to record your resource **connection string** for this quickstart.
 
 ## Code structure
 
-- ./Chat/ClientApp: frontend client
-  - ./Chat/ClientApp/src
-    - ./Chat/ClientApp/src/Components : React components to help build the client app chat experience
-    - ./Chat/ClientApp/src/Containers : Connects the redux functionality to the React components
-    - ./Chat/ClientApp/src/Core : Containers a redux wrapper around the Chat SDK
-    - ./Chat/ClientApp/src/index.js : Entry point for the client app
-- ./Chat/Controllers : Server app core logic for client app to get a token to use with the Azure Communication Services Web Chat SDK
-- ./Chat/Program.cs : Server app program logic
-- ./Chat/Startup.cs: Server app startup logic
+- ./Chat/src/app: Where the client code lives
+- ./Chat/src/app/App.tsx: Entry point into the chat sample 
+- ./Chat/src/app/HomeScreen.tsx: The first screen in the chat sample
+- ./Chat/src/app/ConfigurationScreen.tsx: Where to set your name and avatar for the chat
+- ./Chat/src/app/ChatScreen.tsx: The main chat thread screen
+- ./Chat/src/app/EndScreen.tsx: The screen the user will see when they are no longer in the chat thread
+- ./Chat/src/app/ErrorScreen.tsx: The screen when an error occurs in the chat thread
+- ./Server: server code
+- ./Server/appsettings.json: Where to put your azure communication services connection string
```

Comment:
v. concerned this will lead to customers committing their connection string to source code

Reply:
I am open to any suggestions. We were doing this before and we are trying to port over what we have in our UI Library repo while making it easier to port over any future changes. (e.g we could combine certain files but then we would need to make modifications and then merge them over. The less modifications when porting the less errors we can introduce in the porting process)

Reply:
Going to leave this in for now and resolving this one.